datahub-web-react/src/app/entity/user/preview/Preview.tsx

```diff
@@ -19,19 +19,16 @@ export const Preview = ({
     const entityRegistry = useEntityRegistry();
 
     return (
-        <DefaultPreviewCard
-            url={`/${entityRegistry.getPathName(EntityType.CorpUser)}/${urn}`}
-            title={
-                <Space size="large">
-                    <Avatar size="large" src={photoUrl || defaultAvatar} />
-                    <Space direction="vertical" size={4}>
-                        <Typography.Title style={{ margin: '0', color: '#0073b1' }} level={3}>
-                            {name}
-                        </Typography.Title>
-                        <Typography.Text style={{ color: '#gray' }}>{title}</Typography.Text>
-                    </Space>
+        <Link to={`/${entityRegistry.getPathName(EntityType.CorpUser)}/${urn}`}>
+            <Space size={28}>
+                <Avatar size={60} src={photoUrl || defaultAvatar} />
+                <Space direction="vertical" size={4}>
+                    <Typography.Title style={{ margin: '0', color: '#0073b1' }} level={3}>
+                        {name}
+                    </Typography.Title>
+                    <Typography.Text style={{ color: '#gray' }}>{title}</Typography.Text>
```

Comment:
`#gray`? does that work?

Reply:
Good catch - This should also get extracted out